databases/postgresql-cloudnativepg/manifests/05-replication/my-source-cluster.yaml

```diff
@@ -0,0 +1,98 @@
+# Copyright 2021 Google LLC
```

Comment:
Since these are new files we should have `2023` in this (the copyright header checks are failing due to this)

Reply:
Fixed, thank you, my mistake